### PR TITLE
Updates to the FormService Microservice for FHEO.  Changes to microse…

### PR DESCRIPTION
**Pull request by stephanie-graham**
…rvice in tickets: FAAS-493, FAAS-529, FAAS-530, FAAS-531, FAAS-532, FAAS-533, FAAS-534, FAAS-535, FAAS-536, FAAS-537, FAAS-538, FAAS-541, FAAS-542, & FAAS-545, & FAAS-555

### CODE REVIEW

File: src/modules/home/home.tsx [hunk 1]
```diff
@@ -1,191 +1,475 @@
...
+
+function handleAccordionClick(headerId, contentId){
+  var accordionTopButton = document.getElementById(headerId);
+  var accordionContent = document.getElementById(contentId);
+  if(accordionTopButton && accordionContent){
+    accordionTopButton.setAttribute("aria-expanded", "false");
+    accordionContent.setAttribute("hidden", "true");
+  }
+}
+
```
Comment (Wube-T-Kifle) [inline src/modules/home/home.tsx]: Please convert this function to an arrow function.

File: src/modules/home/home.tsx [hunk 2]
```diff
@@ -1,191 +1,475 @@
-import './home.scss';
-import { useCallback } from 'react';
-import { useHistory } from 'react-router-dom';
+import "./home.scss";
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+
+function handleAccordionClick(headerId, contentId){
+  var accordionTopButton = document.getElementById(headerId);
```
Comment (Wube-T-Kifle) [inline src/modules/home/home.tsx]: Use let keyword. Var variables belong to the global scope.

Comment (stephanie-graham) [reply src/modules/home/home.tsx]: Done, here and in confirmation.tsx

File: src/modules/home/home.tsx (hunk 1 above)
Comment (stephanie-graham) [reply src/modules/home/home.tsx]: done.  Here and in confirmation.tsx.  Please let me know if it doesn't look right.

File: src/modules/confirmation/confirmation.tsx [hunk 3]
```diff
@@ -1,90 +1,317 @@
...
+
+function handleAccordionClick(headerId, contentId){
+  var accordionTopButton = document.getElementById(headerId);
+  var accordionContent = document.getElementById(contentId);
+  if(accordionTopButton && accordionContent){
+    accordionTopButton.setAttribute("aria-expanded", "false");
+    accordionContent.setAttribute("hidden", "true");
+  }
+}
+
```
Comment (Wube-T-Kifle) [inline src/modules/confirmation/confirmation.tsx:12]: convert this to an arrow function

Comment (stephanie-graham) [reply src/modules/confirmation/confirmation.tsx:12]: I think done?  Modified entire function to arrow function.  Did you want me to modify the setAttribute function to an arrow function?

**Pull request merged**